docs/ens.md

```diff
@@ -0,0 +1,104 @@
+
+![Inputs needed to register a KSM address](assets/ens/03-min.png)
+
+After clicking Save, your Ethereum wallet will ask you to confirm a transaction. Once processed, the
+record will show up on the domain's page:
+
+![KSM address now visible in bruno.eth records](assets/ens/04-min.png)
+
+## Adding via the Resolver
+
+DOT has only recently been added to ENS, so it's not available in the UI yet. Let's go through an
+alternative method of registering a DOT address on the `bruno.eth` domain.
+
+First, you need to find out which resolver we have set on the domain. This should be
+[Public Resolver 2](https://etherscan.io/address/0x4976fb03c32e5b8cfe2b6ccb31c09ba78ebaba41), but
+you can double check on the domain's ENS page.
+
+![The domain's resolver is highlighted on the ENS app page](assets/ens/05-min.png)
+
+Next, you need to go to the
+[resolver contract's Write Contract interface](https://etherscan.io/address/0x4976fb03c32e5b8cfe2b6ccb31c09ba78ebaba41#writeContract)
+by clicking on _Contract -> Write Contract_, or appending `#writeContract` to the resolver's URL in
+Etherescan, like so:
+https://etherscan.io/address/0x4976fb03c32e5b8cfe2b6ccb31c09ba78ebaba41#writeContract
+
+There should be an option to "Connect to Web3" which, when clicked, will ask your active wallet for
+permission to connect.
+
+![Connect to Web3 option](assets/ens/06-min.png)
+
+Allow this. If for some reason the option "Connect to Web3" does not automatically switch to
+"Connected - Web3 [YOUR_ADDRESS]", refresh the page and try again.
+
+![Connected to Web3](assets/ens/07-min.png)
+
+Scroll down to `setAddr`. There will be three input fields on this function: `node` is the
+[namehash](https://docs.ens.domains/contract-api-reference/name-processing#algorithm) of your
+domain. You can get it by entering your domain into
+[this tool](https://swolfeyes.github.io/ethereum-namehash-calculator/). Cointype is the ID of the
+supported coin, as specified in the [address-encoder](https://github.com/ensdomains/address-encoder)
+library which wallets can use to integrate ENS. The file to look for is
+[src/index.ts](https://github.com/ensdomains/address-encoder/blob/master/src/index.ts) - scroll to
+the bottom and notice DOT and KSM. The coin ID of KSM is 434, while DOT is 354, so we enter this
+into `coinType`. `a` is the address to bind to this value, in public key format. To turn your
+address into a public key, use [this tool](https://www.shawntabrizi.com/substrate-js-utilities/) and
+enter your address into the `AccountId to Hex` left field, then copy the output of the right field.
+
+![Filled out information in the contract's write interface](assets/ens/08-min.png)
+
+Clicking Write will initiate a transaction. Note that Etherescan's UI is in beta and might throw a
+GasLimit error. If this happens, modify the gas limit in your wallet to something like 10x the
+default. A third or so will get spent, with the rest refunded.
```

Comment:
How much gas does it cost? We should suggest just a little over the required amount because if people send gas limit's of 10x the amount it might impact the time it takes for their transaction to be mined due to filling up blocks.